framework/src/modules/chain/transactions/votes_weight.js

```diff
@@ -224,9 +224,13 @@ const prepare = async (stateStore, transaction) => {
 	];
 
 	const cacheFilter = senderRecipientVotedPublicKeys.map(publicKey => ({
-		address: getAddressFromPublicKey(publicKey),
+		publicKey,
 	}));
 
+	if (cacheFilter.length === 0) {
```

Comment:
shouldn't this check goes before creating filter?

Reply:
@shuse2 what do you mean?

Reply:
I mean maybe we can just check the length of `senderRecipientVotedPublicKeys`?

Reply:
True @michielmulders

Reply:
Yes we can do this as well so we don't have to loop the empty array